.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### The Problem/Feature
+
+99 problems but a PR ain't one
+
+### Implementation details
+
+### Testing Steps
```

Comment:
Add a comment to help them better understand why this heading is here and what should be explained beneath